### ISSUE TITLE
Can I confirm the predicted ages for these sample data?

### ISSUE THREAD

**Issue by wangsl97** [vishnubashyam/DeepBrainNet#7]
Dear,

Can I confirm the predicted ages for these sample data?
I would like to compare your prediction results using the model on sample data to confirm that I have configured this environment correctly.
![prediction_sample_data](https://github.com/vishnubashyam/DeepBrainNet/assets/49799827/e12bc637-41ae-40bd-a975-2b5a674ac1a7)

Thank you.
Best wishes

**Comment by Rachel-672** [comment]
Hi @wangsl97, did you use the bash script to load and make the model work or did you just use google colab? I am also trying to get similar results as the data but they are slightly off

Thank you

**Comment by wangsl97** [comment]
Yes, I run the bash script to load trained model and got the above results. I don't know the real age of these subjects, so I can't check the effect of this model. I really want to get the real age from the author.

**Comment by Rachel-672** [comment]
> Yes, I run the bash script to load trained model and got the above results. I don't know the real age of these subjects, so I can't check the effect of this model. I really want to get the real age from the author.

Is that the information under 'Sample_Data.csv' file in the main page?

**Comment by wangsl97** [comment]
Thank you! I think I forgot to download it on my server. I just checked it on my server before. I get it from homepage of this website.

**Comment by rafaloz** [comment]
I am getting the exact same results... anybody knows why is that differece? Python version? different libraries?

**Comment by wangsl97** [comment]
Sorry, I think there will be something meaningful to explore the difference. Of course, I have no idea on it now.
